src/egraph/add.rs

```diff
@@ -153,7 +154,8 @@ impl<L: Language, N: Analysis<L>> EGraph<L, N> {
         let syn_enode_fresh = syn_enode.apply_slotmap_fresh(&old_to_fresh);
         let i = self.alloc_eclass(&fresh_slots, syn_enode_fresh.clone());
 
-        let syn_app_id = AppliedId::new(i, SlotMap::identity(&syn_enode_fresh.slots()));
+        // #[cfg(feature = "explanations")]
+        // let syn_app_id = AppliedId::new(i, SlotMap::identity(&syn_enode_fresh.slots()));
```

Comment:
oh, is this not used anymore? Feel free to completely remove that line then. No need to have it commented out.